api.go

```diff
@@ -541,6 +545,12 @@ type SearchOptions struct {
 
 	// SpanContext is the opentracing span context, if it exists, from the zoekt client
 	SpanContext map[string]string
+
+	// EvaluateFileTombstones, if enabled, tells searcher to consider a repository's FileTombstones
+	// field when evaluating a search request.
+	//
+	// Note: This feature flag is off by default.
+	EvaluateFileTombstones bool
```

Comment:
I'm unsure if we need to feature flag, given this field won't be set unless we set it. IE once a shard contains tombstones, lets use them!

Reply:
Yeah I still think we should not have the feature flag. If tombstones are present we should read them.